src/components/Project.js

```diff
@@ -33,15 +46,41 @@ class Project extends Component {
     ).start();
   }
 
+  animateHeight(toHeight) {
+    Animated.timing(
+      this.state.slideAnim,
+      {
+        toValue: toHeight,
+        easing: Easing.linear,
+        duration: 300,
+        delay: 200,
+      }
+    ).start()
+  }
 
-  handleClick() {
+  handlePress() {
     GoogleAnalytics.trackEvent('view', this.props.project.display_name)
+
+    const hasSingleMobileWorkflow = length(this.props.mobileWorkflows) === 1
+    const hasMixedWorkflows = length(this.props.mobileWorkflows) > 0 && length(this.props.nonMobileWorkflows) > 0
+
+    if ((hasMixedWorkflows && this.props.promptForWorkflow)) {
+      this.setState({showWorkflowPrompt: true})
+    } else if (hasSingleMobileWorkflow) {
+      //Actions.Classify({ workflowID: head(this.props.mobileWorkflows).id })
```

Comment:
I think Planet Four: Ridges is the only project currently on the app that falls under this condition, and when I click the project, nothing happens. I assume if this condition (`else if (hasSingleMobileWorkflow)`) is removed the workflow will open with `this.openExternalProject()`, so that might be better for now until that swipe PR is merged.